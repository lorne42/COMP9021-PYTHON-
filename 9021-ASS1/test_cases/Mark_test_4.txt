You have met a group of 3 sirs. The Sirs Tracy, Wendy, and Eustice. Sir Eustice says: "Sir Tracy is a Knave." Sir Wendy says: "Sir Eustice is a Knight." Sir Wendy says: "Sir Tracy and I are Knaves." 