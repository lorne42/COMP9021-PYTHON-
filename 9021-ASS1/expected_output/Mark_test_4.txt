The Sirs are: Eustice Tracy Wendy
There is a unique solution:
Sir Eustice is a Knave.
Sir Tracy is a Knight.
Sir Wendy is a Knave.